### ISSUE TITLE
Black screen on Linux desktop conections

### ISSUE THREAD

**Issue by edorbike** [Ylianst/MeshCentral#37]
Hi, I´m traying to connecto to linux ubuntu 18 desktop but black screen showed clicking on connect.

![linuxmesh](https://user-images.githubusercontent.com/45072744/49275704-69072c80-f47c-11e8-980a-9b10eb1ce7a3.jpg)

**Comment by Ylianst** [comment]
Yes, this is a problem with Ubuntu 18.04 and Bryan has been working for a few days on a fix for it. Hopefully we will have a fix next week.

**Comment by Ylianst** [comment]
MeshCentral v0.2.3-w now in NPM should fix this. We can't capture the login screen, but once a user is logged in, it works. The login screen capture seems to be a limitation of the recent Ubuntu 18.04. Please check and report back if it works.

**Comment by edorbike** [comment]
Clicking on the connect button changes the status from disconect to conecting them setup and finally connected but the screen is still black.

**Comment by krayon007** [comment]
I just updated the agent, so now when there is a problem calling into X to get the desktop buffer, it will propagate the error back to the server, which will relay it to the browser, where it will pop up an alert, saying there was a problem calling XOpenDisplay(). This way if the problem happens in the future, we'll know where to look, rather then staring at a black screen scratching our heads :)
